components/yearly_monthly_toggle/index.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState} from 'react';
+
+import SwitchSelector from 'react-switch-selector';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import './index.scss';
+
+interface Props {
+    updatePrice: (isMonthly: boolean) => void;
+}
+
+function YearlyMonthlyToggle({updatePrice}: Props) {
+    const [isMonthly, setIsMonthly] = useState(true);
+    const [toggleBorderClassName, setToggleBorderClassName] = useState('toggle-border');
+
+    const options = [
+        {
+            label: <p className={isMonthly ? 'text text-unselected' : 'text text-selected'}>{'Yearly'}</p>,
+            value: 'Yearly',
```

Comment:
Will want to localize this

Reply:
By localize this, is that just putting 'Yearly' into its own constant?

Reply:
He means adding it to the translations file